styles.css

```diff
@@ -623,3 +632,453 @@ button:disabled {
 }
+
+@media screen and (min-width: 768px) {
+  .header {
+    width: 100vw;
+    height: 72px;
+  }
+
+  .name {
+    margin-left: 12px;
+    font-size: 19px;
+    line-height: 20px;
+    width: 92px;
+    height: 20px;
+  }
+
+  .new-querie {
+    background: url("images/Illustration-header.png");
+    background-position: right;
+    background-repeat: no-repeat;
+    background-size: 48%;
+    width: 100vw;
+    height: max-content;
+    background-color: #1c1a19;
+    align-items: flex-start;
+  }
+
+  .headline {
+    background: none;
+    background-color: #1c1a19;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 24px;
+    width: 39.3vw;
+    height: max-content;
+    margin-left: 143px;
+    border-radius: 0;
+    padding: 192px 0;
+  }
+
+  .introduction {
+    width: 29.3vw;
+    height: max-content;
+    font-family: "crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 76px;
+    letter-spacing: 0.8px;
+    line-height: 82px;
+    color: #f55800;
+    padding: 0;
+    margin-top: 0;
+  }
+
+  .supporting-text {
+    width: 29.3vw;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+    display: flex;
+    align-items: center;
+    color: #fff;
+    margin: 0;
+    padding: 0;
+  }
+
+  .description {
+    width: 29.3vw;
+    font-family: Inter, sans-serif;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    color: #f4f5f7;
+    margin: 0;
+    padding: 0;
+  }
+
+  .positionin {
+    position: absolute;
+    left: 36px;
+    top: 351px;
+  }
+
+  .social-networks-buttons {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    align-self: flex-start;
+    gap: 20px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .works {
+    width: 100vw;
+    height: max-content;
+  }
+
+  .title-content {
+    margin-top: 135px;
+    flex-direction: row;
+    align-items: center;
+    width: 78.68vw;
+    max-height: 52px;
+    margin-bottom: 114px;
+  }
+
+  .title-works {
+    width: 315px;
+    height: 52px;
+    color: #091e42;
+    margin: 0;
+    padding: 0;
+    border-spacing: 0.1px;
+    font-size: 39px;
+  }
+
+  .solidhr {
+    width: 55.13vw;
+  }
+
+  .feature-card {
+    flex-direction: row;
+    align-items: flex-start;
+    width: 80.27vw;
+    height: auto;
+  }
+
+  .image-feature {
+    width: 47.5vw;
+    height: max-content;
+  }
+
+  .card2 {
+    gap: 24px;
+    width: 31.11vw;
+    height: max-content;
+  }
+
+  .multi-post {
+    font-size: 40px;
+    line-height: 54px;
+  }
+
+  .description2 {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .card-works {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
+    place-content: center;
+    justify-content: center;
+    row-gap: 37px;
+    column-gap: 24px;
+    height: max-content;
+    width: 80.2vw;
+  }
```

Comment:
- [ ] Kindly correct the title of each card to be similar to the Figma design.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/213598478-46feb16f-7418-426a-ae52-9cd1680500e1.png) | ![image](https://user-images.githubusercontent.com/58642949/213598520-c7f380f4-52db-4d09-a171-7568cbee8745.png) |